README.rst

```diff
@@ -276,11 +276,11 @@ Choose from the list of available rules:
 
   Configuration options:
 
-  - ``multiLineExtendsEachSingleLine`` (``bool``): whether definitions should be
-    multiline; defaults to ``false``
-  - ``singleItemSingleLine`` (``bool``): whether definitions should be single line
-    when including a single item; defaults to ``false``
-  - ``singleLine`` (``bool``): whether definitions should be single line; defaults
+  - ``multi_line_extends_each_single_line`` (``bool``): whether definitions should
```

Comment:
Should aliases be documented?

Reply:
yes, look at `around_argument_assignments` and follow the approach